migrate/createdatabase.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE IF NOT EXISTS `community_voices` DEFAULT CHARACTER SET latin1 COLLATE latin1_swedish_ci;
```

Comment:
See run-db.sh. 

This is just needed because the sql.dump which overrides schema.sql will not include 'CREATE DATABASE or USE DATABASE' and thus when running startup scripts the user will not have a database created.